staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller_reconcile.go

```diff
@@ -501,13 +502,23 @@ func (c *policyController) latestPolicyData() []policyData {
 				if definitionInfo.lastReconciledValue.Spec.ParamKind != nil {
 					hasParam = true
 				}
+				matchConditions := definitionInfo.lastReconciledValue.Spec.MatchConditions
+				matchExpressionAccessors := make([]cel.ExpressionAccessor, len(matchConditions))
+				for i, condition := range matchConditions {
+					matchExpressionAccessors[i] = &matchconditions.MatchCondition{
+						Name:       condition.Name,
+						Expression: condition.Expression,
+					}
```

Comment:
nit: I think you should be able to just do this here
```suggestion
					matchExpressionAccessors[i] = (*matchconditions.MatchCondition)(&matchConditions[i])
```

Reply:
Definitely true after type aliasing. Would it have been true before then? That'd be pretty cool.

Reply:
wait, might not be true, these are technically different types... will test.

Reply:
Ah, it does work, MatchCondition is type aliased